atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/AbstractSnapshotTransactionTest.java

```diff
@@ -3650,4 +3684,9 @@ private static List<Cell> generateCells(int numberOfRowsPerTable, int numberOfCe
                         Cell.create(PtBytes.toBytes("row" + (i % numberOfCellsPerRow)), PtBytes.toBytes("column" + i)))
                 .collect(Collectors.toUnmodifiableList());
     }
+
+    private enum UncommittedValueType {
```

Comment:
might as well extract to a separate test class file if we're using in more than one test class?

Reply:
Yep that makes sense!